packages/components/src/StepperWizard/types.ts

```diff
@@ -0,0 +1,7 @@
+import { type ReactNode } from 'react';
+
+export type Steps = {
```

Comment:
```suggestion
export type Step = {
```

в StepperWizardProps:
steps: Array<Step>;

И выносить тут в отдельный файл с типами кажется излишним

Reply:
Steps это тип для массива шагов, вынес его в отдельный файл с типами, для возможности использовать его при создании шагов